penpost/penpost-web
# Gaj/feature/static login

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 

This adds a static login page with basic logic for checking user login.

Issues closed #18 

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

NA

> Describe your code changes in detail for reviewers.

Login component was included to prepare for application of authentication. For now, is basic and will allow a user to login with any combo of username or password as long as both are more than 6 characters long.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

See `components/Login` for styling and logic. 

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Please run `npm start` on this branch and check that in `App` that Login is imported and is being called. 

### Steps To Take
> Any questions and possible steps moving forward.

Apply Route to Login.
